### PR TITLE
Bug/wrong error in pep authentication

### PR DESCRIPTION
**Pull request by dmoranj**
Fix issue #182 
Fix issue #180

### CODE REVIEW

File: lib/errors.js [hunk 1]
```diff
@@ -54,11 +54,6 @@ var errors = {
         this.message = 'The system wasn\'t able to parse the given JSON payload (either it was empty or wrong)';
         this.code = 400;
     },
-    WrongXmlValidationResponse: function() {
-        this.name = 'WRONG_XML_VALIDATION_RESPONSE';
-        this.message = 'The system wasn\'t able to parse the XML obtained from the Access Control';
-        this.code = 500;
-    },
```
Comment (jcalderin) [inline lib/errors.js]: This error has to be removed in the file "errorcodes.md"

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -11,4 +11,6 @@
 - Group and redefine error codes.
 - Add execution mode without Access Control authorization for the PEP Proxy (#173)
 - Check the service and subservice headers for content (#176)
-- Fix the Keystone user authentication error was raised as a connection one (#174)
\ No newline at end of file
+- Fix the Keystone user authentication error was raised as a connection one (#174)
+- Fix type of error when authenticating PEP Proxy (#182)
+- Fix some errors appear in the API documentation an don't ever occur (#180)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: an -> and

Comment (dmoranj) [inline CHANGES_NEXT_RELEASE]: Fixed in 81af5d9

File: lib/errors.js (hunk 1 above)
Comment (dmoranj) [inline lib/errors.js]: Fixed in 81af5d9

**Comment by jcalderin** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**